### PR TITLE
chore: sql query optimizations

### PR DESCRIPTION
**Pull request by MohamadJaara**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

sql query optimizations for the following tables:
1. Calls
2. Clients
3. Assets

### Causes (Optional)

_Briefly describe the causes behind the issues. This could be helpful to understand the adopted solutions behind some nasty bugs or complex issues._

### Solutions

_Briefly describe the solutions you have implemented for the issues explained above._

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: persistence/src/commonMain/db_user/com/wire/kalium/persistence/Calls.sq [hunk 1]
```diff
@@ -67,3 +91,6 @@ SET status = 'CLOSED'
 WHERE
     status != 'CLOSED'
     AND status != 'MISSED';
+
+callerIdByConversarionId:
+SELECT caller_id FROM Call WHERE conversation_id = :conversationId LIMIT 1;
```
Comment (alexandreferris) [inline persistence/src/commonMain/db_user/com/wire/kalium/persistence/Calls.sq:96]: Couldn't find where this is being used? 🤔

**Review by alexandreferris — COMMENTED** [review]
Great work @MohamadJaara !

Just have a single comment on where a SQL query is being used as I couldn't find it 🤔

File: persistence/src/commonMain/db_user/com/wire/kalium/persistence/Calls.sq [hunk 2]
```diff
@@ -9,9 +9,9 @@ CREATE TABLE Call (
     caller_id TEXT NOT NULL,
     conversation_type TEXT AS ConversationEntity.Type NOT NULL,
     created_at TEXT NOT NULL,
-
     PRIMARY KEY (conversation_id, id)
 );
+CREATE INDEX CallDateIndex ON Call (created_at);
```
Comment (vitorhugods) [inline persistence/src/commonMain/db_user/com/wire/kalium/persistence/Calls.sq:14]: ⭐⭐⭐⭐⭐

**Review by alexandreferris — APPROVED** [review]
👌🏻

**Pull request merged**